scripts/integration_test.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -euox pipefail
+
+# Script that run the whole data pipeline the fastest possible to validate
+# that every part is working with the others
+
+# Supposed to be run from the repository root directory
+
+cd http_service/models/;
+
+# Remove the model
+rm defectenhancementtaskmodel* || true;
+rm backout* || true;
+
+# First retrieve a subset of bugs data
+# TODO: Let the script download the previous DB as it should be pretty fast?
+bugbug-data-bugzilla --limit 100
+
+# Then retrieve a subset of commit data
+mkdir -p cache
+bugbug-data-commits --limit 100 cache
+
+# Then train a bug model
+bugbug-train --limit 500 --no-download defectenhancementtask
+
+# Then train a commit model
+bugbug-train --limit 30000 --no-download backout
+
+# Then spin up the http service up
+# This part duplicated the http service Dockerfiles because we cannot spin up Docker containers on
```

Comment:
Here is the answer I got from the taskcluster IRC channel: 
```
privileged is only allowed on some workers, since it's basically root on the hsot
the more common solution for that is to run the task on a dedicated, non-docker worker
```

Reply:
We have the `privileged` capability on the relman-svc worker, if you need it (indeed, we are using it for the docker_build task).
If you just need `dind` instead, it's available everywhere (e.g. https://github.com/rail/taskboot-test/blob/5c4e0994f28d261e64aaf06d13dbd4d36d83aa05/.taskcluster.yml#L37-L38).

Reply:
I think (but I squashed my branch now) that I've hit the issue where the docker server is too old and I needed the privileged capability for mounting the docker socket.

I doubt that I can use the relman-svc worker for the bugbug PR and as we will need to run the integrations tests when tagging, we are likely stuck in the current situation.